packages/amplify_analytics_pinpoint/android/build.gradle

```
@@ -67,11 +67,11 @@ dependencies {
     api amplifyCore
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'com.amplifyframework:aws-analytics-pinpoint:1.24.1'
-    implementation 'com.amplifyframework:aws-auth-cognito:1.24.1'
-    testImplementation 'junit:junit:4.13.2'
-    testImplementation 'org.mockito:mockito-core:3.10.0'
-    testImplementation 'org.mockito:mockito-inline:3.10.0'
-    testImplementation 'androidx.test:core:1.4.0'
+    implementation 'com.amplifyframework:aws-analytics-pinpoint:1.25.1'
+    implementation 'com.amplifyframework:aws-auth-cognito:1.25.1'
+    testImplementation 'junit:junit:4.13'
+    testImplementation 'org.mockito:mockito-core:3.1.0'
+    testImplementation 'org.mockito:mockito-inline:3.1.0'
+    testImplementation 'androidx.test:core:1.2.0'
```

Test deps should match the above

```suggestion
    testImplementation 'junit:junit:4.13.2'
    testImplementation 'org.mockito:mockito-core:3.10.0'
    testImplementation 'org.mockito:mockito-inline:3.10.0'
    testImplementation 'androidx.test:core:1.4.0'
```